packages/mold/build.sh

```diff
@@ -2,19 +2,11 @@ TERMUX_PKG_HOMEPAGE=https://github.com/rui314/mold
 TERMUX_PKG_DESCRIPTION="mold: A Modern Linker"
 TERMUX_PKG_LICENSE="AGPL-V3"
 TERMUX_PKG_MAINTAINER="@termux"
-_COMMIT=ce744aeca423954c9152a6c1c2185f149bf50ad8
-TERMUX_PKG_VERSION=1.0.3.20220212
-TERMUX_PKG_REVISION=2
+TERMUX_PKG_VERSION=1.1.1
 TERMUX_PKG_SRCURL=https://github.com/rui314/mold.git
```

Comment:
Any reason to still check this out from git, rather than use [the official source tarfile](https://github.com/rui314/mold/releases/tag/v1.1.1)?